src/cuttlefish.app.src

```diff
@@ -1,15 +1,11 @@
-{application, cuttlefish,
- [
-  {description, "cuttlefish configuration abstraction"},
-  {vsn, git},
-  {registered, []},
-  {applications, [
-                  kernel,
-                  stdlib
-                 ]},
-  {env, []},
-
-  {maintainers,[]},
-  {licenses,["Apache"]},
-  {links,[{"Github","https://github.com/tsloughter/cuttlefish"}]}
- ]}.
+{application, cuttlefish, [
+    {description,   "cuttlefish configuration abstraction"},
+    {vsn,           git},
+    {registered,    []},
+    {applications,  [kernel, sasl, stdlib, syntax_tools, getopt, lager]},
+    {env,           []},
+    {modules,       []},
+    {maintainers,   ["RabbitMQ Team @ Pivotal Software"]},
+    {licenses,      ["Apache 2.0"]},
+    {links,         [{"GitHub",  "https://github.com/rabbitmq/cuttlefish"}]}
```

Comment:
was wrong before too but that I look at it now this should probably point to Kyorai